src/cmd/linuxkit/pkglib/pkglib.go

```diff
@@ -69,7 +69,7 @@ func NewFromCLI(fs *flag.FlagSet, args ...string) ([]Pkg, error) {
 	// Defaults
 	piBase := pkgInfo{
 		Org:          "linuxkit",
-		Arches:       []string{"amd64", "arm64", "s390x"},
+		Arches:       []string{"amd64", "arm64", "s390x", "riscv64"},
```

Comment:
This makes it a _default_ for building, but not enable it. Can you check if your riscv builds work even without this change? As I see the logic, this should not get in the way.

This really should go somewhere more explicit, but that is beyond the scope of this PR.

Reply:
You were right -- so far I think it is fine to leave it as-is. It is confusing tho